indigo_app/templates/indigo_app/user_popup.html

```diff
@@ -0,0 +1,23 @@
+<div class="popup-content lh-compact">
```

Comment:
Please move this template and the corresponding views and urls into `indigo_social`, so it lives with the other social-related code.